mppt/240.OutputEnable.uavcan

```diff
@@ -0,0 +1,16 @@
+#
+# Service to enable or disable the mppt output
+#
+
+void6
+
+# If both enable and disable are true then no change is applied
+bool enable
```

Comment:
why have two bool values? why not just the enable value, and set to false to dissable?

Reply:
if it is needed, then a comment explaining why we have both would be nice

Reply:
This was given to me by the manufacturer. I didn't have any input on it's design. I totally agree that it's a less than ideal method. Meanwhile, the company has shipped hardware using this.

Reply:
I've re-worked this. It's much more verbose now